fedimint-client/src/module/mod.rs

```diff
@@ -805,7 +809,7 @@ pub trait IClientModule: Debug {
 
     fn input_fee(&self, amount: Amount, input: &DynInput) -> Option<Amount>;
 
-    fn output_fee(&self, output: &DynOutput) -> Option<Amount>;
+    fn output_fee(&self, amount: Amount, output: &DynOutput) -> Option<Amount>;
```

Comment:
I see we have the amount in `input_fee` too (I think because we already removed some data from the DB when this fn is called, so might not always know), but why can't the client figure out itself how much an output is worth?

Reply:
True, not all inputs contain their amount but all outputs do. It was just a reflex on my part to mirror the input_fee. I would leave it like that for now and at some point remove the input_fee and output_fee altogether in favor of adding a fee field to ClientInput and ClientOutput just as we did with the amount previously. Already had started to work on the refactor and it was significantly cleaner but good stuck on the lnv1 fees as they are so weird.